landoapi/api/landings.py

```diff
@@ -118,6 +121,13 @@ def update(landing_id, data):
         revision (sha) of push if landed == true
         empty string if landed == false
     """
+    if request.headers['API-Key'] != TRANSPLANT_API_KEY:
+        return problem(
+            401,
```

Comment:
this should be 403; 401 means "unauthenciated" and requires a www-authenticate header in the response.